Marajoara.Cinema.Management/Marajoara.Cinema.Management.Api/Controllers/CineRoomController.cs

```diff
@@ -11,7 +11,8 @@
 namespace Marajoara.Cinema.Management.Api.Controllers
 {
     [ApiController]
-    [Authorize(Roles = "Manager")]
+    [Authorize]
+    //[Authorize(Roles = "Manager")]
```

Comment:
Não sei se é melhor deixar o manger no controller ou por rota.
Mas se for por rota acho que da para remover esse comentário e o [Authorize]

Reply:
Removi o comentário. 
Como os gets são utilizados na tela de new ticket, precisei deixar com permissão para todos os roles;

já as demais ações, deixei como manager mesmo.